src/pages/course-page/CoursePage.tsx

```diff
@@ -43,16 +56,22 @@ const CoursePage = () => {
 
   useEffect(() => {
     const handleKeyDown = (event: KeyboardEvent) => {
-      if (event.key === '1' && video?.playbackRate) {
-        event.preventDefault();
-        if (video?.playbackRate < 2) {
-          video.playbackRate += 0.1;
-        }
-      } else if (event.key === '0' && video?.playbackRate) {
-        event.preventDefault();
-        if (video?.playbackRate > 0.5) {
-          video.playbackRate -= 0.1;
-        }
+      event.preventDefault();
+
+      if (
```

Comment:
Maybe these  `if` statements should be util functions like `changePlaybackRate` that accept `event` and `videoRef` as props and do some logic?